shoes-core/lib/shoes/common/translate.rb

```diff
@@ -0,0 +1,19 @@
+class Shoes
+  module Common
+    module Translate
+      def translate_left
+        left, = translate
+        left || 0
+      end
+
+      def translate_top
+        _, top = translate
+        top || 0
+      end
```

Comment:
Seeing that translate it something that we need to use on pretty much every draw, how about setting it as some instance variables? I might pre prematurely optimizing things here but the access seems a bit weird this way and it's just set one time isn't it?